api_specifications/case-api-0.1.0.yaml

```
@@ -627,6 +682,23 @@ components:
           type: string
           description: The outcome of the case. This can be a judgement, a settlement or
             any other outcome.
+        pendingAdvocateRequest:
+          type: array
+          items:
+            properties:
+                advocateId:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 14 but found 16

(indentation)